src/main/java/com/google/sps/data/Chat.java

```diff
@@ -58,38 +58,64 @@ public static Chat getChat(Datastore datastore, String chatid) {
     }
   }
 
-  public static List<Chat> getChats(Datastore datastore, String participantid) {
+  public static List<Chat> getUserChats(Datastore datastore, String participantid) {
     
     List<Chat> chats = new ArrayList<>();
-    Query<Entity> chatQuery = Query
-    .newEntityQueryBuilder()
-    .setKind("Chat")
-    .build();
+    Query<Entity> chatQuery = Query.newEntityQueryBuilder().setKind("chat").build();
     QueryResults<Entity> chatEntities = datastore.run(chatQuery);
 
-    // Iterate through the chats
-    // for (Entity entity : chatEntities.asIterable()) {
-    //   long chatid = entity.getKey.getId();
-    //   List<String> participantList = Arrays.asList(entity.getProperty("participants").split(","));
+    while(chatEntities.hasNext()){
+        Entity entity = chatEntities.next();
+        String participants = entity.getString("participants");
+        List<String> participantList = Arrays.asList(participants.split(","));
```

Comment:
```suggestion
        List<String> participantList = entity.getList("participants");
```

A quick Google search of "datastore entity getList" led me to the [`getList`](https://googleapis.dev/java/google-cloud-datastore/latest/com/google/cloud/datastore/BaseEntity.html#getList-java.lang.String-) method.